components/Navbar/mobile/MobileMenu.tsx

```diff
@@ -0,0 +1,49 @@
+'use client';
+
+import { motion } from 'framer-motion';
+import React from 'react';
+import NavMenuItems from '../NavMenuItems';
+import CloseIcon from './CloseIcon';
+import DownloadIcon from './DownloadIcon';
+
+const MobileMenu = ({
+  menuState: { isMenuOpen, toggleMenu },
+}: {
+  menuState: { isMenuOpen: boolean; toggleMenu: () => void };
+}) => {
+  return (
+    <motion.div
+      initial={{ y: 100, opacity: 0 }}
+      animate={{ y: 0, opacity: 1 }}
+      transition={{
+        duration: 0.3,
+        ease: 'easeOut',
+      }}
+      className='absolute right-[0.5px] top-2 z-20 w-[95vw] bg-white-900'
+    >
+      <div className='flex flex-col gap-6 p-4'>
+        <div className='flex w-full justify-end'>
+          <button onClick={toggleMenu}>
+            <CloseIcon />
+          </button>
+        </div>
+        <div className='flex-between flex-col gap-2'>
+          <NavMenuItems />
+        </div>
+
+        <a
+          className='flex-center alt-btn gap-2 sm:text-black'
+          href='/'
+          download='resume'
+        >
+          <div className='self-center'>
+            <DownloadIcon />
+          </div>
+          Resume
+        </a>
```

Comment:
Let's add a hover effect here to make it clear we're highlighting a button.

Reply:
Do we need a hover state for mobile elements?

Reply:
Those elements show up at <640px width. Not exclusively mobile devices. Checkout a repo on github and bring your window width to about 600px you'll see some small screen specific Star/Fork buttons with hover effects.

I agree it may not be seen, but let's add it in case someone with a 1280x1024 monitor views your site while splitting the screen.

![image](https://github.com/jacastanon01/portfolio/assets/4108484/d36cd0f6-3021-471b-bcf5-c78ed7be15d7)

Reply:
Good to know, thanks for the explanation